### ISSUE TITLE
Sidebar scroll arrows are missing tooltips

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#21474]
## Description 

Sidebar scroll arrows are missing tooltips

## Steps to Reproduce

   1. install `1.38.23`
   2. launch Brave
   3. load a bunch of websites
   4. add each one to the Sidebar, via the `Add to Sidebar` hover
   5. notice you get scroll arrows appearing for overflow
   6. hover over the scroll arrows

## Actual result:

No tooltip

## Expected result:

Tooltip

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.38.23 Chromium: 99.0.4844.51 (Official Build) nightly (x86_64)
-- | --
Revision | d537ec02474b5afe23684e7963d538896c63ac77-refs/branch-heads/4844@{#875}
OS | macOS Version 11.6.3 (Build 20G415)

cc @rebron @simonhong
